Makefile

```diff
@@ -35,6 +35,7 @@ docs/$(RELEASE_LATEST).tgz: $(CHART)/Chart.yaml $(CHART)/templates/*.yaml
 	helm package build/$(CHART_LATEST) -d docs
 
 clean:
+	rm -rf docs/$(RELEASE).tgz docs/$(RELEASE_LATEST).tgz docs/es/all.yaml docs/es/all-latest.yaml
```

Comment:
this doesn't seem quite right to me.  There are definitely times you want to do this step--like when you have just built something but want to force rebuild it--but you also don't want to do "make clean" and remove the last release.

I think we should consider the use case for when you want to get a new build, and why that isn't working when source are updated.  There's probably some dependency missing in the Makefile specification for how those artifacts should be updated.

Reply:
added clean-all command